src/ProtocolGateway.Core/ProtocolGateway.Core.csproj

```diff
@@ -32,6 +32,24 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
```

Comment:
Service Fabric requires 64 bit, I changed all libraries that will work with anycpu back to anycpu.